dms_field/models/dms_file_template.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class DmsFileTemplate(models.Model):
+    _name = "dms.file.template"
```

Comment:
Why `dms.file.template` instead of `dms.field.template`? The other name is misleading.